src/components/molecules/Settings/Account/AccountSection/index.tsx

```diff
@@ -74,6 +74,14 @@ const ProfileSection: React.FC<Props> = ({
     [updatePassword],
   );
 
+  const handleThemeUpdate = useCallback(
+    (theme?: string) => {
+      if (!theme) return;
+      updateTheme?.(theme as Theme);
```

Comment:
Areas where bugs are likely to occur: Please avoid using "as" if at all possible, as is just "as" dangerous as any.

Reply:
I would have done that, but the handleThemeUpdate is being used and not the updateTheme because it is being passed into the `EditableItem` component, which expects its `onSubmit` to be `(body: string) => void`.

Reply:
Do you want me to update the EditableItem in this PR? I'm not great at generics so it might take a little while for me to sort it out. Also, we use that component in many sections of the Settings.